index.html

```diff
@@ -574,6 +576,145 @@ <h3 class="heading2 newsletter__heading">
             </div>
         </section>
 
+        <!-- Footer -->
+
+        <div class="container footer-container">
+            <footer class="footer">
+                <div class="footer__content">
+                    <div class="footer__texts">
+                        <div class="footer__brand">
+                            <div class="footer__logo">
+                                <img
+                                    src="/assets/images/logo.svg"
+                                    alt="logo image"
+                                />
+                            </div>
+                            <div class="heading6 footer__heading">Travlog</div>
+                        </div>
+                        <p class="body2 footer__body">
+                            Contrary to popular belief, Lorem Ipsum is not
+                            simply random text. It has roots in a piece of
+                            classical Latin literature from 45 BC.
+                        </p>
+                    </div>
+                    <div class="footer__socials">
+                        <a href="/" class="footer__social-link">
+                            <span
+                                class="icon-facebook footer__social-link--facebook"
```

Comment:
```suggestion
                                class="icon-facebook"
```

why have we aded extra modifier?